### ISSUE TITLE
I successfully installed WSA, but couldn't open it

### ISSUE THREAD

**Issue by MirionQs** [LSPosed/MagiskOnWSA#348]
I opened the task manager and found that WSA closed after a short flash

**Comment by PeterNjeim** [comment]
Not enough information. Please share logs from the Event Viewer → Windows Logs → Application. Show us both the General tab and the Details tab. Make sure you selected the correct event in the Even Viewer, it should have the same time as when the crash occurred.

**Comment by noahhhi** [comment]
run powershell with admin
 `cd "your WSA file location"`
`set-executionpolicy -executionpolicy unrestricted`
or just simply set (**UNSAFE**)
`Set-ExecutionPolicy -ExecutionPolicy Unrestricted CurrentUser`

@yujincheng08 The guide need to be updated.

**Comment by PeterNjeim** [comment]
@noahhhi You simply need to run the ps1 script, it will auto-ask for admin permissions, and auto-sets the execution policy. What does your comment have to do with this issue?

**Comment by noahhhi** [comment]
PowerShell scripts have blocked by default on some devices. There's no
permission windows pop-up unless you enable it.

https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_execution_policies?view=powershell-7.2

**Comment by noahhhi** [comment]
@PeterNjeim PowerShell scripts have blocked by default on some devices. There's no permission windows pop-up unless you enable it.

https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_execution_policies?view=powershell-7.2

**Comment by PeterNjeim** [comment]
Did you not answer my question on purpose? I asked what your complaint had to do with this issue

**Comment by noahhhi** [comment]
@PeterNjeim I am sorry that I copy the answer to the wrong issue🌚

**Comment by PeterNjeim** [comment]
Also your claim is wrong. The instructions state to "Run with PowerShell". This isn't affected by the default security policy: https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_run_with_powershell?view=powershell-7.2.

I suggest you don't:
- hijack random issues with irrelevant (and incorrect) information
- mention (@) maintainers with incorrect information

Please open a new issue or pull request if you believe your problem is real

**Comment by noahhhi** [comment]
[image: image.png]
install.ps1 file instantly closes when ran w/ powershell · Issue #59 ·
LSPosed/MagiskOnWSA (github.com)
<https://github.com/LSPosed/MagiskOnWSA/issues/59>
install.ps1执行即闪退 · Issue #117 · LSPosed/MagiskOnWSA (github.com)
<https://github.com/LSPosed/MagiskOnWSA/issues/117>
Running the Install.ps1 file via Powershell does nothing · Issue #143 ·
LSPosed/MagiskOnWSA (github.com)
<https://github.com/LSPosed/MagiskOnWSA/issues/143>
Install.ps autocloses on Newest Beta Channel · Issue #298 ·
LSPosed/MagiskOnWSA (github.com)
<https://github.com/LSPosed/MagiskOnWSA/issues/298>
